packages/blocks/src/api/raw-handling/code-handler.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { mapValues } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import onFilter from './on-filter';
+import scriptRemover from './script-remover';
+import { deepFilterHTML } from './utils';
+import {
+	getBlockType,
+	getFreeformContentHandlerName,
+	getUnregisteredTypeHandlerName,
+} from '../registration';
+import { parseWithGrammar } from '../parser';
+
+export function codeHandler( { HTML = '' } ) {
+	const freeformName = getFreeformContentHandlerName();
+	const unregisteredName = getUnregisteredTypeHandlerName();
+
+	return parseWithGrammar( HTML ).map( ( block ) => {
+		const { name } = block;
+
+		// Let TinyMCE handle it
```

Comment:
Is this still necessary? Maybe we should just do it for all blocks if we only strip script and on* attributes. The freeform block could be something other than TinyMCE.

Reply:
My thinking was to really minimise back-compat issues, and that may or may not include preserving even those tags. However,

> The freeform block could be something other than TinyMCE.

I did wonder about this. I considered the alternative, which was to hardcode `if ( name === 'core/freeform' )`... Worse? Better?

Reply:
I'm just not sure why it should be an exception. Great if TinyMCE already does some sanitising, but it doesn't hurt to run our own?